playwright/visual-test/TagData-dark.spec.mjs

```diff
@@ -0,0 +1,8 @@
+// @flow strict
```

Comment:
I'm seeing a vertical black line on the left side of the remove click affordance. I'm also seeing a similar vertical white  line on the middle example in the light mode image (which may also be present in the left example — I can't tell because it's white-on-white, where it should be using a gray fill). I assume those aren't intentional.

Reply:
Yeah this was weird. If you happen to visit the Visual-Test website, it seems like they don't show up. But when taking a snapshot, they do. 

If you visit the website, you can't see it:

https://deploy-preview-2963--gestalt.netlify.app/visual-test/TagData

but in the snapshot, I'm seeing this line thing:

<img width="390" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/5b2a8c62-b4eb-4203-a7bf-3531e1409bf1">

Reply:
Have you generated new snaps after the recent changes? Is the vertical line still present?

Reply:
Great, regenerating seemed to work